index.html

```diff
@@ -489,20 +489,24 @@
           <a href='https://www.w3.org/TR/ldp/#ldpc-HTTP_DELETE'>section 5.2.5</a>, it must also follow the additional
           behavior outlined below.
         </p>
-        <section id="httpDELETEDepth">
-          <h3>Depth Header</h3>
+        <section id="httpDELETERecursion">
+          <h3>Recursive Delete</h3>
```

Comment:
I would add a non-normative note that the spec is written from the perspective that clients will assume from an advertised DELETE that they do not have to look for contained/child resources to clean up.